test/mocks/StakingMock.sol

```diff
@@ -0,0 +1,60 @@
+pragma solidity 0.4.18;
+
+import "@aragon/apps-staking/contracts/interfaces/IStaking.sol";
+
+
+contract StakingMock is IStaking {
```

Comment:
Just a note: Love how this nice mock allows for easier unit testing of PLCR. We will need to do proper integration testing for all the user stories once we put everything together to ensure it all works as expected with real staking!